docs/book/language-reference.md

```diff
@@ -111,6 +111,7 @@ complex types.
 | Built-in | Inputs | Description |
 | ------- |--------|-------------|
 | <span class="opa-keep-it-together">``io.jwt.verify_rs256(string, certificate, output)``</span> | 1 | ``output`` is ``true`` if the RS256 signature of the input token is valid. ``certificate`` is the PEM encoded certificate used to verify the RS256 signature|
+| <span class="opa-keep-it-together">``io.jwt.verify_hs256(token, secret, output)``</span> | 1 | ``output`` is ``true`` if the Secret signature of the input token is valid. ``secret`` is a plain text secret used to verify the HS256 signature|
```

Comment:
[nitpick] This would be a tad more in line with the surrounding docs if it was

```diff
- io.jwt.verify_hs256(token, secret, output)
+ io.jwt.verify_hs256(string, secret, output)
```

Reply:
Good point, added a commit to address this.